neutron/neutron.go

```diff
@@ -30,12 +30,13 @@ const (
 
 // NetworkV2 contains details about a labeled network
 type NetworkV2 struct {
-	Id                string   `json:"id"` // UUID of the resource
-	Name              string   // User-provided name for the network range
-	SubnetIds         []string `json:"subnets"`         // an array of subnet UUIDs
-	External          bool     `json:"router:external"` // is this network connected to an external router
-	AvailabilityZones []string `json:"availability_zones"`
-	TenantId          string   `json:"tenant_id"`
+	Id                  string   `json:"id"` // UUID of the resource
+	Name                string   // User-provided name for the network range
+	SubnetIds           []string `json:"subnets"`         // an array of subnet UUIDs
+	External            bool     `json:"router:external"` // is this network connected to an external router
+	AvailabilityZones   []string `json:"availability_zones"`
+	TenantId            string   `json:"tenant_id"`
+	PortSecurityEnabled *bool    `json:"port_security_enabled"`
```

Comment:
PortSecurityEnabled is a *bool because it's optional depending on whether you are using the ML2 extension to neutron.  If it's nil, you don't aren't using the extension.  External is not optional.